docs/install/cli.md

```diff
@@ -0,0 +1,62 @@
+# Installing the smctl
+
+To use the Service Manager CLI you need to download and install it first:
+
+## Approach 1: Get the latest Service Manager CLI release
+
+You can get started with the CLI by simply downloading the latest release from [HERE][2].
+
+## Approach 2: Manual installation
```

Comment:
this doesn't work

Reply:
I copied this from the sm cli docs. Will remove it